Makefile

```diff
@@ -1,12 +1,15 @@
+export FLASK_APP=app
```

Comment:
If you merge `master` I think these changes will go away for this PR.

Reply:
Will do before final merge